bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/dto/clip2/Resources.java

```diff
@@ -29,7 +28,11 @@ public class Resources {
     private List<Resource> data = new ArrayList<>();
 
     public List<String> getErrors() {
-        return errors.stream().map(Error::getDescription).collect(Collectors.toList());
+        return errors.stream().map(Error::getDescription).toList();
```

Comment:
Could we please keep the old Collectors.toList() syntax here? I am still maintaining a back port to OH v3.x..

Reply:
This is not just syntax. See https://rules.sonarsource.com/java/RSPEC-6204/

In general I consider this topic already covered by https://github.com/openhab/openhab-addons/pull/15419#discussion_r1292739917 and https://github.com/openhab/openhab-addons/pull/15390#discussion_r1292741086.

Feel free to continue backporting, of course. I don't know how you approach it from a practical perspective, but it might make your life easier if you consider `main` as upstream to your 3.x branch, i.e. cherry-pick commits into it and then apply Java and openHAB version compatibility patches after that when needed.

Reply:
The prior code worked fine. There is no reason to change it.

Reply:
PS I know that I have no influence in this discussion since you have always the power to overrule. But if you do insist on overruling then I think you should at least be self consistent and change ALL the cases in my code where I have used Collectors.toList .. there is no point in being half pure.

Reply:
I was applying the "boy scout rule" here. However, I have now reverted it from this PR since this refactoring was only related by "other changes close by". This can definitely be fixed more consistently in a follow-up PR and you can simply omit that resulting commit from your backporting efforts.